demo/vite.config.ts

```diff
@@ -60,6 +61,7 @@ export default defineConfig((config) => ({
 	plugins: [
 		copySamples(),
 		includeSamples(),
+		listSamples(),
```

Comment:
We don't need the plugin in main vite.config for the moment (it has zero impact, but just to be cleaner).

Reply:
Indeed it is not needed in the main vite.config. I have removed it.